tn_api_v2_core.yml

```diff
@@ -0,0 +1,956 @@
+                    type: string
+                    example: "fb3d8eff432a329dc21fad8449c686fa93e5ce24"
+                datasource_responses:
+                    description: The list of datasource responses to be used to generate solutions.
+                    type: object
+                    allOf:
+                        -   $ref: '#/components/schemas/FullDatasourceResponse'
+
+        MinimalDatasourceResponse:
+            type: object
+            properties:
+                4f499298b303c55a1ee522118afce28abe2f68e4:
+                    description: The ID of the datasource request. The ID varies between requests.
+                    type: array
+                    items:
+                        $ref: "#/components/schemas/BasicGenerateDatasourceRequest"
+
+        FullDatasourceResponse:
+            type: object
+            properties:
+                4f499298b303c55a1ee522118afce28abe2f68e4:
+                    description: The ID of the datasource request. The ID varies between requests.
+                    type: array
+                    items:
+                        $ref: "#/components/schemas/FullGenerateDatasourceRequest"
+
+        DatasourceSolution:
+            type: object
+            properties:
+                price:
+                    type: float
+                    description: Price of the segment (fare and tax included).
+                    example: 197.1
+                base_price:
+                    type: float
+                    description: Base price of the segment (tax excluded)
+                    example: 0
+                taxes:
+                    type: float
+                    description: Tax price of the segment
+                    example: 119.1
+                fees:
+                    type: float
+                    description: Fees for the segment, if applicable.
+                    example: 0
+                latest_ticketing_date:
+                    type: date
+                    description: Latest ticketing time
+                    example: "2019-05-07"
+                refundable:
+                    type: boolean
+                    description: Refundable tickets
+                    example: false
+                transportation_time:
+                    type: integer
+                    description: Total transportation time for the itinerary
+                    example: 425
+                segments:
+                    type: array
+                    description: List of segments in the itinerary
+                    items:
+                        type: object
+                        description: Segment details
+                        properties:
+                            segment_id:
+                                type: string
+                                description: This is the same as the segment_id in the segment_details response.
```

Comment:
What is the segment_details response? 

I was thinking that the customer would have to provide this field, and they need to ensure it will be unique within the response as a whole. As each segment will need a unique ID i think.

Reply:
oops, copying and pasting does not always work